package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
```

Comment:
I personally set it with `npm config set loglevel silent` and change it as needed to higher levels, but I don't have a strong opinion on this.

It could be worth noting also that maybe even if you set -s here, if you don't have it when you do `npm start` you will probably get the obnoxious message if the error code is >0 (I haven't checked tho).

I say let's leave it like this, if it becomes annoying I'm happy to review or submit a patch to change it. Would that be ok?